README.md

```diff
@@ -1,17 +1,17 @@
 Numix icon theme
 ================
-Numix is the official icon theme from the [Numix Project](http://numixproject.org). It is heavily inspired by, and based upon parts of the Elementary, Humanity and Gnome icon themes. Designed to be used along side the application icon theme [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle). This readme provides information on [installation](https://github.com/numixproject/numix-icon-theme#installation) and [icon requests](https://github.com/numixproject/numix-icon-theme#icon-requests). Licensed under the GPL-3.0+
+Numix is the official icon theme from the [Numix Project](http://numixproject.org). It is heavily inspired by, and based upon parts of the Elementary, Humanity and Gnome icon themes. Numix is designed to be used along-side an application icon theme like [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle) or [Numix Square](https://github.com/numixproject/numix-icon-theme-square). This readme provides information on [installation](https://github.com/numixproject/numix-icon-theme#install-it) and [icon requests](https://github.com/numixproject/numix-icon-theme#icon-requests). Licensed under the GPL-3.0+
+
+## Install it
```

Comment:
@Foggalong I'm guessing because that's what we're using in the gtk theme :wink: :